server/README.md

```diff
@@ -26,4 +25,6 @@ be up and running
 
 - `--url`: Set the url of the server you are running on (this is only required for the frontend) (Default: `http://localhost:8080`)
 - `--port`: Port to start the server on (Default: `8080`)
-- `--local`: Use the assets (frontend & blacklist) directly from the filesystem instead of the internal binary storage. This helps during the development of the frontend as you do not have to do `go generate ./...` after every change. This should not be used in production. (Default: `false`)
\ No newline at end of file
+- `--local`: Use the assets (frontend & blacklist) directly from the filesystem instead of the internal binary storage. This helps during the development of the frontend as you do not have to do `go generate ./...` after every change. This should not be used in production. (Default: `false`)
+- `--blacklist-sources`:  Comma separated list of blacklist urls to periodically sync (Default https://hosts.ubuntu101.co.za/domains.list)
+- `--sync`: Blacklist synchronization interval (Default: one hour)
```

Comment:
Please add info about the interval time. What does e.g. `--sync 60` mean? 60 milliseconds/seconds/minutes/hours ?

Reply:
Ah found the format in the command help. `XhXmXs` please add it here in the documenttion as well